proxy/ParentSelection.h

```diff
@@ -59,12 +59,7 @@ enum ParentResultType {
   PARENT_FAIL,
 };
 
-enum ParentRR_t {
-  P_NO_ROUND_ROBIN = 0,
-  P_STRICT_ROUND_ROBIN,
-  P_HASH_ROUND_ROBIN,
-  P_CONSISTENT_HASH,
-};
+enum ParentRR_t { P_NO_ROUND_ROBIN = 0, P_STRICT_ROUND_ROBIN, P_HASH_ROUND_ROBIN, P_CONSISTENT_HASH, P_LATCHED_ROUND_ROBIN };
```

Comment:
Oops, I'll fix that.  

The interfaces are defined in ParentSelection.h and the enum is used in both ParentSelection.cc, ParentConsistentHash.cc and ParentRoundRobin.cc all of which include ParentSelection.h